### ISSUE TITLE
`release_bullets()` for future releases?

### ISSUE THREAD

**Issue by AshesITR** [r-lib/lintr#1292]
What `release_bullets()` should we define for future (> 3.0.0) release issues of lintr?

CC @MichaelChirico @hadley for input.

**Comment by hadley** [comment]
What release tasks do you currently have to manually remember to do?

**Comment by MichaelChirico** [comment]
@hadley this is both of our first release of `lintr`, so I think we're both figuring it out as we go along :)

I think we should always do a .dev/compare_branches of old CRAN vs latest dev on a sizeable chunk of packages -- that's pretty much our revdeps since we don't expect many downstream packages to be on CRAN (maybe as Suggests?)

It would be good to have a nice source of scripts to get a sense of how things change outside of packages too. I will try and run internally at Google.

**Comment by hadley** [comment]
Hmmm, and you can't even really use revdepcheck, since `.lintr` won't be included in CRAN packages.

**Comment by AshesITR** [comment]
True (revdepcheck returns no problems), but do you know if the CRAN sources include it?
Then we could at least run on all packages that Suggest lintr.

**Comment by MichaelChirico** [comment]
CRAN sources won't have anything .Rbuildignore'd:

https://github.com/r-lib/lintr/blob/19308c1d61dae2a9cc921a9cbd921c1df31c867e/.Rbuildignore#L13

I suggest we look for GitHub URLs in our reverse suggests and run compare_branches for all of them too to try and give a heads up about (1) upcoming breaking changes and (2) upcoming delta of lints for their repo

**Comment by MichaelChirico** [comment]
oh that's right, `available.packages()` doesn't include the `URL` key of the package's DESCRIPTIONs -- @hadley is there a tool to retrieve those that you're aware of? if not we can scrape the canonical CRAN pages...

**Comment by AshesITR** [comment]
@MichaelChirico You can use `read.dcf()` with an open URL file:

```r
f <- file("https://cran.r-project.org/web/packages/ggplot2/DESCRIPTION", open = "r")
read.dcf(f, "URL")[[1L]]

#> [1] "https://ggplot2.tidyverse.org,\nhttps://github.com/tidyverse/ggplot2"
```

**Comment by MichaelChirico** [comment]
What's discussed here is seen in .dev/revdep*. We can add more incrementally later, not sure I see a reason to keep this open now.
